.script/tests/KqlvalidationsTests/CustomTables/GitHubAuditData.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
is this file change required. If not remove this change

Reply:
required for validation added by @v-sabiraj